test/addons_test.go

```diff
@@ -65,7 +66,7 @@ func TestAddons(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	testAddons := []v1beta1.AddonInterface{}
+	testAddons := make([]v1beta1.AddonInterface,0)
```

Comment:
> idiomatic empty initialization is to use var s []string rather than s := []string{} because it actually saves a potentially redundant allocation
```suggestion
	var testAddons []v1beta1.AddonInterface
```

Reply:
TIL. I was just about to comment the opposite.